pkgs/by-name/hu/hugo/package.nix

```
@@ -48,10 +48,12 @@ buildGoModule rec {
     version = "v${version}";
   };
 
-  meta = with lib; {
+  meta = {
+    changelog = "https://github.com/gohugoio/hugo/releases/tag/v${src.rev}";
```

```suggestion
    changelog = "https://github.com/gohugoio/hugo/releases/tag/v${version}";
```
`src.rev` is `"refs/tags/v${version}"`, so using it for the changelog won't work.